README.md

```diff
@@ -16,6 +16,9 @@ A module for provisioning an IBM Cloud Red Hat OpenShift cluster on VPC Gen2. Th
 - Make sure that you have a recent version of the [IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cli-getting-started)
 - Make sure that you have a recent version of the [IBM Cloud Kubernetes service CLI](https://cloud.ibm.com/docs/containers?topic=containers-kubernetes-service-cli)
 
+## Addons
```

Comment:
Why do you think we need this section in the main readme file? The release notes would include information that this has been added and the example readme file should include this kind of information.

Reply:
If you do think the information is important for users, I would add something to the main description under the title.

Reply:
Yea, the release note has the information on the feature. Also the example readme mentions that this example showcases autoscaling.